### ISSUE TITLE
Western blotting exp tech bug - bands appear over the edge of the blot

### ISSUE THREAD

**Issue by ald32** [starteam/starcellbio_html#340]
When I have 15 samples (and no marker) in the western blotting experimental technique, the 15th sample extends over the edge of the blot when it should be contained within the confines of the blot.

**Comment by annagav** [comment]
To test this in EX3, select fifteen samples and for blotting conditions select 'Mouse anti-PGK1', secondary antibody 'rabbit anti-mouse'.

**Comment by annagav** [comment]
![screen shot 2016-10-26 at 2 12 19 pm](https://cloud.githubusercontent.com/assets/7574259/19738490/ac9e1bc2-9b85-11e6-8f24-10efb4f2d569.png)

**Comment by annagav** [comment]
@ald32, @laleman Are we supposed to see the marks if the marker was not loaded?

**Comment by laleman** [comment]
Yes, you are supposed to see the protein bands, but not the sizes of the proteins which is indicated with numbers on the right of the gel. The picture you took is exactly what we are suppose to see.

**Comment by annagav** [comment]
Try to shift the bands to the left.
